libwallet/assets/dcr/ticket.go

```diff
@@ -204,7 +197,22 @@ func (asset *Asset) VSPTicketInfo(hash string) (*VSPTicketInfo, error) {
 
 	ticketInfo.Client = vspClient
 
-	vspTicketStatus, err := vspClient.GetTicketStatus(ctx, ticketHash)
+	txs, _, err := asset.Internal().DCR.GetTransactionsByHashes(ctx, []*chainhash.Hash{ticketHash})
+	if err != nil {
+		return nil, err
+	}
+
+	commitmentAddr, err := stake.AddrFromSStxPkScrCommitment(txs[0].TxOut[1].PkScript, asset.chainParams)
```

Comment:
The now-removed `vspClient.GetTicketStatus` method called the private `vspClient.status` method which performed some additional checks after getting the tx but these checks aren't performed here:
- Check if `len(ticketTx.TxOut)` is 3
- Check the tx against `stake.IsSStx`

Reply:
Resolved